backend/app/api/api_v1/routers/admin.py

```diff
@@ -198,57 +200,32 @@ async def import_law_policy(
             StringIO(law_policy_csv.file.read().decode("utf8"))
         )
         valid_metadata = get_valid_metadata(db)
+        existing_import_ids = [id[0] for id in db.query(Document.import_id)]
 
         encountered_errors = {}
         document_create_objects: list[DocumentCreateRequest] = []
 
+        import_ids_to_create = []
         # TODO: Check for document existence?
         for validation_result in extract_documents(
             csv_reader=csv_reader, valid_metadata=valid_metadata
         ):
             if validation_result.errors:
                 encountered_errors[validation_result.row] = validation_result.errors
             else:
+                import_ids_to_create.append(validation_result.import_id)
                 document_create_objects.append(validation_result.create_request)
```

Comment:
we could be skipping the import ids that already exist here, rather than in the background task...

Reply:
Let me do that